### DISCUSSION TITLE
Pycaret with Parallel Backend

### DISCUSSION THREAD

**Discussion by RahulDas-dev** [pycaret/pycaret#3609] [Q&A]
"Could you please provide a minimal example demonstrating the usage of a parallel backend? I was unable to find any relevant information or examples regarding parallel backends in the tutorials directory."

**Comment by tyler-corcoran** [comment]
Also looking for the same. I see there was an overhaul listed on the road map for 3.0.0. Is this native or are there steps to initiate the solutions proposed?

**Comment by ngupta23** [comment]
Can you try with this: https://towardsdatascience.com/scaling-pycaret-with-spark-or-dask-through-fugue-60bdc3ce133f
This article was writtent for non-time series pycaret module, but in theory the same calls should work for time series as well. If it does not work, feel free to open a GitHub Issue for it.

**Comment by ngupta23** [comment]
You can also check out this documentation on the same: https://fugue-tutorials.readthedocs.io/tutorials/integrations/ecosystem/pycaret.html. Let me know if any of this works.

**Comment by ngupta23** [comment]
Here is another comprehensive example of how I have used Ray, Spark, and Fugue to benchmark pycaret time series across multiple datasets (> 3000).

https://github.com/pycaret/benchmarks/blob/main/benchmarks/parallel/execution.py

**Reply by jmakov** [reply]
It would help very much if this kind of examples would be documented in pycaret.
